library/std/src/sys_common/wtf8.rs

```diff
@@ -983,3 +986,56 @@ impl Hash for Wtf8 {
         0xfeu8.hash(state)
     }
 }
+
+/// A safe iterator over a LPWSTR
+/// (aka a pointer to a series of UTF-16 code units terminated by a NULL).
+pub struct WStrUnits<'a> {
```

Comment:
WStrUnits doesn't do anything related to utf8/wtf8, so can you rename the module to `wstr` or `utf16` or something like that?